### PR TITLE
Workaround Windows issues

### LINKED ISSUE #2399: [UNDER-DISCUSSION] Internal network issues
--------
Under Discussion Topic
--------

Internal networking issues are preventing windows from cloning SRAW.

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with the under_discussion type?
- [x] 2. If implemented, it will add a new requirement?
- [x] 3. Is a rationale provided? (Such as explaining why the improvement is needed )

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. The discussion determined the addition of a new ```task``` issue?

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address?
Closes #2399 

##### What are the significant changes in functionality due to this change request?
This works around two windows issues.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: rook/main.py [hunk 1]
```diff
@@ -68,6 +68,8 @@ class NoColors:
 parser = argparse.ArgumentParser(description="Test Runner")
 parser.add_argument('-j', '--jobs', dest='number_jobs', type=int, default=1,
                     help='Specifies number of tests to run simultaneously (default: 1)')
+parser.add_argument('--longest-jobs',dest='longest_jobs', type=int, default=0,
+                    help='Only for compatibility')
```
Comment (wangcj05) [inline rook/main.py:72]: Thanks for add this option, is it already used in our regression testing?

**Review by wangcj05 — APPROVED** [review]
changes are good.

**Comment by wangcj05** [comment]
checklist is good, and PR can be merged.

File: rook/main.py (hunk 1 above)
Comment (joshua-cogliati-inl) [reply rook/main.py:72]: Civet uses it to "Print the longest running jobs upon completion" We could add an actual feature to do this if we want.

**Pull request merged**